functions/notifyDeliveryInProgress/source.js

```diff
@@ -100,28 +100,36 @@ exports = async function (changeEvent) {
       mo.Message.Body.Html.Data = body;
       return mo;
     }
-  
-    try { 
+
+    try {
         // A driver was assigned and the status updated to IN PROGRESS
-        if ( operationType === "update" && 
-             changeEvent.ns.coll === "orders" && 
+        if ( operationType === "update" &&
+             changeEvent.ns.coll === "orders" &&
              updateDescription.updatedFields.status &&
              fullDocument.status === "IN PROGRESS" &&
              fullDocument.assignedToDriver &&
              fullDocument.emailAddress ) {
-          
+
                 // TODO: Call a function to create a completion url.
                 // Build requester message
-                let { emailAddress, address, zipcode, items, firstName, lastName } = fullDocument;
+                let { emailAddress, address, zipcode, items, firstName, lastName, assignedToOrg } = fullDocument;
+                let query = {assignedToOrg: String(assignedToOrg)}
```

Comment:
is the correct query for the organizations collection?

Reply:
I haven't tested this yet but it should be correct.

Reply:
{_id: String(assignedToOrg)}

maybe? Not positive though.